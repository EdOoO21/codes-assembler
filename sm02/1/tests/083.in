728 754 -59 -819 -738 -225 -930 -900 950 330 -566 -613 -644 -368 -831 676 -889 -160 -686 -278 -383 148 662 -586 -27 609 282 448 845 900 -530 682 -87 977 -659 -910 600 -938 -960 981 550 -45 -285 -209 175 481 -619 127 908 788 -351 -23 -69 631 921 783 221 735 -342 -98 -529 -722 865
