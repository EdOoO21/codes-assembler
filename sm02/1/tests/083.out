865
-722
-529
-98
-342
735
221
783
921
631
-69
-23
-351
788
908
127
-619
481
175
-209
-285
-45
550
981
-960
-938
600
-910
-659
977
-87
682
-530
900
845
448
282
609
-27
-586
662
148
-383
-278
-686
-160
-889
676
-831
-368
-644
-613
-566
330
950
-900
-930
-225
-738
-819
-59
754
728
